conf/prod.routes

```diff
@@ -1,5 +1,6 @@
 # Add all the application routes to the app.routes file
-->         /api-notification-pull              app.Routes
+->         /                          app.Routes
 ->         /                          health.Routes
 
+GET        /api/definition            @uk.gov.hmrc.apinotificationpull.controllers.DefinitionController.get
```

Comment:
You're missing the RAML route, also it's possibly a good idea to pull it out into a definition.routes like

https://github.com/hmrc/api-subscription-fields/blob/master/conf/definition.routes

Reply:
We wouldn't normally need to place this route in the RAML as it is only for the purpose of the platform to configure the API and is not exposed externally. Have moved out into a separate definition.raml.